integration-cli/docker_cli_help_test.go

```diff
@@ -27,9 +23,10 @@ func TestMainHelpWidth(t *testing.T) {
 	for _, line := range lines {
 		if len(line) > 80 {
 			t.Fatalf("Line is too long(%d chars):\n%s", len(line), line)
+
 		}
 		if home != "" && strings.Contains(line, home) {
-			t.Fatalf("Line should use ~ instead of %q:\n%s", home, line)
+			t.Fatalf("Line should use home shortcut instead of %q:\n%s", home, line)
```

Comment:
updated this by putting this functionality into pkg/homedir. please review again.